Changelog.md

```diff
@@ -2,6 +2,18 @@
 
 ## From 2.x Librarian-Puppet requires Ruby >= 1.9, uses Puppet Forge API v3. For Ruby 1.8 use 1.x
```

Comment:
If we're no longer testing with 1.9, should we officially drop it and make it a 3.0? I've seen builds pass with this so I'd be happy with leaving it in.

Reply:
I think that's sending the right message. Let me update the message and the version.